programming/sr/motors/index.md

```diff
@@ -81,29 +79,28 @@ When you set the motor power to 0, this signals the Motor Board to actively stop
 
 ~~~~~ python
 # store the motor in a local variable because typing it out gets really boring
-molly = R.motors[0].m0
+molly = R.motor_boards["srABC1"].motors[1]
```

Comment:
anecdote: the name "molly" came from a team which had labelled their `m0` and `m1` outputs "molly" and "milly", so the choice of `m0` here was very much intentional.